src/components/Map.js

```diff
@@ -0,0 +1,22 @@
+import React, {Component} from "react"
+import {View} from "react-native"
+import styles from "../styles"
+
+export default class SizeAwareComponent extends Component {
```

Comment:
Something weird going on here? Every other class is defined in a file with the same name. I'd just say "Rename this class to Map?" but the implementation does seem to match this name more than a map.